### PR TITLE
Created Usage setting article

### PR DESCRIPTION
**Pull request by jonat123**
New article explaining the new Usage tab in the settings menu for umbraco Cloud projects

### CODE REVIEW

File: Umbraco-Cloud/Set-Up/Usage/index.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+versionFrom: 7.0.0
+---
+
+# Usage on your Umbraco Cloud project
+
+In the Umbraco Cloud settings tab, we recently introduced a new setting called Usage.
+
+From here you can see your current usage of Content nodes, Custom hostnames and Media storeage and the usage limitations for your Umbraco Cloud project and the plan that the project is on.
```
Comment (sofietoft) [inline Umbraco-Cloud/Set-Up/Usage/index.md:9]:
```suggestion
From here you can see your current usage of Content nodes, Custom hostnames, and Media storage. You can also get an overview of the usage limitations for your Umbraco Cloud project as well as the plan that the project is on.
```

File: Umbraco-Cloud/Set-Up/Usage/index.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+---
+versionFrom: 7.0.0
+---
+
+# Usage on your Umbraco Cloud project
+
+In the Umbraco Cloud settings tab, we recently introduced a new setting called Usage.
```
Comment (sofietoft) [inline Umbraco-Cloud/Set-Up/Usage/index.md:7]: It won't be a new page in half a year 😉 
Perhaps consider not calling it *new*.

File: Umbraco-Cloud/Set-Up/Usage/index.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+From here you can see your current usage of Content nodes, Custom hostnames and Media storeage and the usage limitations for your Umbraco Cloud project and the plan that the project is on.
+
+![Usage on Cloud](images/Usage.png)
+
+## Usage limits
+
+On the image below you can see the Usage limitatons for the specific plans on Umbraco Cloud.
+
+![Usage limits on a starter plan](images/plan_limitations.png)
```
Comment (sofietoft) [inline Umbraco-Cloud/Set-Up/Usage/index.md:17]: Remember to double-check that your images show up on Github. It's very case sensitive 🙈 

```suggestion
![Usage limits on a starter plan](images/Plan_limitations.png)
```

File: Umbraco-Cloud/Set-Up/Usage/index.md (hunk 2 above)
Comment (jonat123) [reply Umbraco-Cloud/Set-Up/Usage/index.md]: Thanks i have now changed it a bit 😄

File: Umbraco-Cloud/Set-Up/Project-settings/index.md [hunk 4]
```diff
@@ -52,6 +52,10 @@ From the *Settings* menu you can upgrade your project to a standard or a Profess
 
 The option will not be available if you are already on the specific plan or if you are running in Trial mode.
 
+## [Usage](../Usage/)
+
+On your Umbraco Cloud project we've made it possible to see the usage of content nodes, custom hostnames and media storage for your project and if it is using above or below the allowed amount for the plan that your project is on.
```
Comment (sofietoft) [inline Umbraco-Cloud/Set-Up/Project-settings/index.md:57]:
```suggestion
On your Umbraco Cloud project, we've made it possible to see the usage of content nodes, custom hostnames and media storage for your project. You are also able to check if it is using above or below the allowed amount for the plan that your project is on.
```

**Pull request merged**